package.json

```diff
@@ -8,6 +8,9 @@
     "prepare": "husky install",
     "test": "echo 'test'"
   },
+  "workspaces": [
+    "packages/falso"
+  ],
```

Comment:
this is a npm@7+ feature, on `npm install` in root will automatically install also packages for `packages/falso`